app/script/auth/component/ClientItem.tsx

```diff
@@ -72,7 +72,11 @@ class ClientItem extends React.Component<CombinedProps, State> {
     validationError: null,
   };
 
-  formatId = (id = '?') => id.toUpperCase().replace(/(..)/g, '$1 ');
+  formatId = (id = '?', outputLength = 16) => {
```

Comment:
This functionality can be moved to our `commons` package.

Reply:
As agreed f2f, will use `String.prototype.padStart()` instead.